packages/storybook/src/example-pages/toepassingen/aanvragen/omgevingsoverleg.content.ts

```diff
@@ -0,0 +1,148 @@
+import { DefinitionList, FormGroupRadios } from "@dso-toolkit/sources";
+import { html, TemplateResult } from "lit-html";
+
+export const definitionList1: DefinitionList = {
+  useSrOnlyColon: false,
+  definitions: [
+    {
+      term: "Verzoek 1",
+      descriptions: [
+        {
+          content: "Object Laan van Eik en Duinen 125, 's-Gravenhage 1",
+        },
+      ],
+    },
+    {
+      term: "Indienen bij",
+      descriptions: [
+        {
+          content: "Hoogheemraadschap van Delfland",
+        },
+      ],
+    },
+    {
+      term: "Soort",
+      descriptions: [
+        {
+          content: "Aanvraag vergunning",
+        },
+      ],
+    },
+    {
+      term: "Bevat",
+      descriptions: [
+        {
+          content: "Milieubelastende activiteit - Vergunning (Gemeente)",
+        },
+      ],
+    },
+    {
+      term: "Omgevingsoverleg mogelijk",
+      descriptions: [
+        {
+          content: "Nee",
+        },
+      ],
+    },
+  ],
+};
+
+export const definitionlist2: DefinitionList = {
+  useSrOnlyColon: false,
+  definitions: [
+    {
+      term: "Verzoek 2",
+      descriptions: [
+        {
+          content: "Object Laan van Eik en Duinen 125, 's-Gravenhage 2",
+        },
+      ],
+    },
+    {
+      term: "Indienen bij",
+      descriptions: [
+        {
+          content: "Gemeente Den Haag",
+        },
+      ],
+    },
+    {
+      term: "Soort",
+      descriptions: [
+        {
+          content: "Melding",
+        },
+      ],
+    },
+    {
+      term: "Bevat",
+      descriptions: [
+        {
+          content: "Milieubelastende activiteit - Vergunning (Gemeente)",
+        },
+      ],
+    },
+    {
+      term: "Omgevingsoverleg mogelijk",
+      descriptions: [
+        {
+          content: "Ja",
+        },
+      ],
+    },
+  ],
+};
+
+export const radios: FormGroupRadios<TemplateResult> = {
+  group: "radios",
+  id: "omgevings-overleg-radio",
+  label: "Wilt u uw verzoeken definitief of als Omgevingsoverleg indienen?",
+  inline: true,
+  info: {
+    fixed: true,
+    active: true,
+    richContent: html`
```

Comment:
Moeten we deze property ook omzetten naar `content` en dat de rich conten wordt meegegeven?

Reply:
mee eens